news/2021-04-06-mappers-report-march.md

```diff
@@ -0,0 +1,173 @@
+- [Guideline regarding sudden drops in hitsound volume](https://osu.ppy.sh/community/forums/topics/1278789)
+
+### osu!catch
+
+- [osu!catch RC rework (part 1)](https://osu.ppy.sh/community/forums/topics/1265691)
+
+## <a id="within-bn-nat"></a>Within the Beatmap Nominators and NAT
+
+As with every other month, we are once again here to show you which poor souls decided to sell themselves for the better. Here is a list of all recent Beatmap Nominator movements. 
+
+### BN Additions
+
+- **osu!:** [VINXIS](https://osu.ppy.sh/users/4323406), [AJT](https://osu.ppy.sh/users/3181083), [Sylas](https://osu.ppy.sh/users/3906405) & [Kudosu](https://osu.ppy.sh/users/11038155)
+- **osu!taiko:** [Horiiizon](https://osu.ppy.sh/users/8071438), [gevbiivi5](https://osu.ppy.sh/users/7140680) & [-Aku](https://osu.ppy.sh/users/12315824)
+- **osu!catch:** [NyalettOwO x3](https://osu.ppy.sh/users/7592136) & [wonjae](https://osu.ppy.sh/users/5032045)
+- **osu!mania:** [Paran](https://osu.ppy.sh/users/13340203), [Komirin](https://osu.ppy.sh/users/4725379) & [AHHHHHHHHHHHHHH](https://osu.ppy.sh/users/8249608)
+
+Do you like modding and helping other people with their beatmaps? Do you want to see your own name on this list? If your answer to both of these questions is "yes", then you can also become a Beatmap Nominator!
+
+The applications are always open, so head over to the [Beatmap Nominator website](https://bn.mappersguild.com/) and apply today!
+
+### BN Departures
+
+- **osu!:** [Lafayla](https://osu.ppy.sh/users/5312547), [riffy](https://osu.ppy.sh/users/597957), [Battle](https://osu.ppy.sh/users/4037545) & [BaAR_Vendel](https://osu.ppy.sh/users/8679346)
+- **osu!taiko:** [frukoyurdakul](https://osu.ppy.sh/users/7612550)
+- **osu!catch:** [GIGACHAD](https://osu.ppy.sh/users/11081858), [Phob](https://osu.ppy.sh/users/6069462) & [Secre](https://osu.ppy.sh/users/2306637)
+
+### NAT Additions
+
+A new red-name appeared out of nowhere! [Zelq](https://osu.ppy.sh/users/8953955) has decided to join the red crusade after two years as a BN and will support the NAT from now on by ensuring that the constant flow of Beatmap Nominators won't come to an end. Once again, welcome to the team Zelq!
+
+What's better than one new red-name? Of course, two of them! [Agatsu](https://osu.ppy.sh/users/5579871) has ascended to the NAT after roughly one and a half years within the BNs. With the same mission as Zelq, Agatsu has rolled up his sleeves to work alongside the NAT to better the mapping and modding community. Welcome to the team Agatsu!
+
+## BN and QAT/NAT Badges
+
+Last but not least, it's been a while since we last time included the badges in the Mappers' Report. Therefore, the list will be a little longer this time! They all received their badge for their longstanding contributions as a BN or a member of the QAT or NAT to their respective game mode. We are thankful for all their contributions and hope to see much more in the future!
```

Comment:
```suggestion
It has been quite a while since we last covered recent badge additions in the Mappers' Report, so bear with us as the list will be a little bit longer this time.

All of those listed below have been longstanding contributors to the Beatmap Nominators or the NAT at large, and we're very thankful for their efforts and drive to help keep osu! awesome.
```